prime-router/src/main/kotlin/SettingsProvider.kt

```diff
@@ -44,19 +44,28 @@ enum class CustomerStatus {
  * A submission with topic FULL_ELR will be processed using the full ELR pipeline (fhir engine), submissions
  * from a sender with topic COVID_19 will be processed using the covid-19 pipeline.
  */
-enum class Topic(val json_val: String) {
+enum class Topic(val jsonVal: String, val isUniversalPipeline: Boolean) {
```

Comment:
Maybe default this to true since we likely won't be adding more topics for the the covid pipeline?